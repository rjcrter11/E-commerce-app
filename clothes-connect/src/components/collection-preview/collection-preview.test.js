import React from 'react';

import { create } from 'react-test-renderer';

import CollectionPreview from './collection-preview.component';

describe('CollectionPreview component', () => {
    let wrapper;

    const mockProps = {
        title: 'hats',
        items: []
    }
    wrapper = create(<CollectionPreview {...mockProps} />)

    it('should render CollectionPreview component', () => {
        expect(wrapper.toJSON()).toMatchSnapshot()
    })
})