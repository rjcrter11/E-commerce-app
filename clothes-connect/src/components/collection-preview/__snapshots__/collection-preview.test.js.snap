// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionPreview component should render CollectionPreview component 1`] = `
<div
  className="collection-preview"
>
  <h1
    className="title"
  >
    HATS
  </h1>
  <div
    className="preview"
  />
</div>
`;
